tests/unit/utils/Utils.h

```diff
@@ -41,10 +42,7 @@ SGStringList<T> generateRandomStringData(
 		    min_string_length;
 		SGVector<T> current(len);
 		/* fill with random uppercase letters (ASCII) */
-		for (index_t j = 0; j < len; ++j)
-		{
-			current[j] = (T)(std::rand() % ('Z' - 'A' + 1) + 'A');
-		}
+		random::fill_array(current, 'A', 'Z', prng);
```

Comment:
does this include 'z'? :)

Reply:
Yup. For some reason, "uniform_int_dist" accepts inclusive range [a,b], while "uniform_real_dist" is [a,b)

Reply:
is that something we came up with, or someone else? :)
Because usually, I would generally expect the int version to be open as well, e.g.
https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.random.randint.html

Reply:
Nope, I just followed how std::uniform_int_dist does it
https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution